qcodes/tests/drivers/keysight_b1500/b1500_driver_tests/test_b1520a_cmu.py

```diff
@@ -226,8 +226,8 @@ def test_run_sweep(cmu):
     cmu.adc_coef(5)
     cmu.run_sweep()
     mainframe.write.assert_has_calls([
-        call('WDCV 3,1,-1.0,1.0,5'),
-        call('WDCV 3,1,-1.0,1.0,5'),
```

Comment:
I am not sure how this ever worked before. The value of 1.0 for end is set in the second call and should therefor not be used in the first call. Rather the default value of 0.0 should be used. Similar for the value of 5.0 which is set in the third call

Reply:
> I am not sure how this ever worked before

me neither :( but great catching all of this!

Reply:
Earlier implementation was definitely wrong. I had missed it and the test passed by fluke. Now, this is correct. Thanks for catching.